### DISCUSSION TITLE
Omap files missing in File Search

### DISCUSSION THREAD

**Discussion by gianvigalta** [OpenOrienteering/mapper#1940] [Questions & Answers] [answered]
Hello,
i have a problem because the OMAP files created by OOM arent' shown in my file explorer. 
I downloaded also other file explorer (Explorer++, FreeCommander) but no way that omap files appears.
They are regular listed if from OOM i click on open project.
![image](https://user-images.githubusercontent.com/15347218/114308277-8be18b00-9ae3-11eb-9f91-077192f438b8.png)

![image](https://user-images.githubusercontent.com/15347218/114308306-a4ea3c00-9ae3-11eb-84b7-917ad7de8c0f.png)

i have done also some registry key work, but noway to get the files.
any suggestion?
with other files any problem at all

**Comment by dg0yt** [comment, marked as answer]
It is strange indeed.
- Which versions of Mapper and Windows do you use?
- What kind of drive is D:?
- Does it also happen on drive C:?
- Did you try to change the Explorer options to show hidden files and system files? (We don't touch these attributes, but your D: might be strange.)

**Reply by gianvigalta** [reply]
Tks for your fast reply
Windows 10 and OOM 0.9.5
D: is internal HardDrive
Same happens on C:
Yes, hidden files on, all on, I try also the service to stop and restart the file search indexing.
Really strange, really. 
The shame is that I can't use those file with other programs, as purplePen

**Reply by dg0yt** [reply]
I have no idea in this case. I don't want to suggest you migh be looking on the wrong folder.

As a next step, I would suggest to visit the Mapper program with Windows Explorer and verify that the examples and symbol sets are visible.

**Reply by gianvigalta** [reply]
Ciao
yes, they are visible no problem and the folfer is ok where i save the files.
Today i tried another way to save the file on a USB key. well, OOM say that the USB Key is writing protected and impossible to write on it, if i do a normal copy/paste of a file from file explorer i con do it no problem.
Mistery continue ...

**Reply by dg0yt** [reply]
Is this some kind of company-managed computer? It sound like Mapper would run in a sandbox, with restricted access to USB, and writes redirected from the original directory. (I'm not aware of a software which does that for local apps, but Softgrid/Microsoft Application Virtualization applies or applied such mechanisms.)

**Reply by gianvigalta** [reply]
No, it's my normal laptop. When I install OOM, my Comodo anti-virus ask if I want to run it on sandbox or free. I click free.
I installed also the OOM older versions but same result.
I will reinstall with Comodo disabled and let you know. 
Tks for your time and help.

**Reply by dg0yt** [reply]
Summarizing from #1942: 
The problem is related to the Comode anti-virus product which includes a sandbox. The product has configuration options to tune this.
